src/mocks/films.js

```diff
@@ -39,20 +97,75 @@ const getBgSrc = (title) => {
 
 const getImgSrc = (movie) => `img/${transformToKebabCase(movie)}.jpg`;
 
-const movies = movieTitles.slice(0, MOCK_MOVIES_AMOUNT).map((title, i) => {
+const getRandomInteger = (min, max) => Math.round(Math.random() * (max - min) + min);
+
+const getRandomItem = (items) => items[getRandomInteger(0, items.length - 1)];
+
+const getRandomSubList = (items, value) => {
+  let subList = [];
+  if (value) {
+    subList = items.slice();
+    for (let i = 0; i < items.length - value; i++) {
+      subList.splice(getRandomInteger(0, subList.length - 1), 1);
+    }
+  }
+  return subList;
+};
+
+const getActorsText = () => {
+  const actors = getRandomSubList(ACTORS, ACTORS_COUNT);
+  return `${actors.join(`, `)} and other`;
+};
+
+const sentences = DESCRIPTION_TEXT.split(`. `).map((item) => item.endsWith(`.`) ? item : `${item}.`);
+
+const getFilmDescription = () => {
+  const paragraphsCount = getRandomInteger(PARAGRAPH_COUNT_MIN, PARAGRAPH_COUNT_MAX);
+  const descriptions = new Array(paragraphsCount)
+    .fill(``)
+    .map(() => {
+      const descriptionLength = getRandomInteger(PARAGRAPH_SENTENCES_MIN, PARAGRAPH_SENTENCES_MAX);
+      return getRandomSubList(sentences, descriptionLength).join(` `);
+    });
+  return descriptions;
+};
+
+const allmovies = movieTitles.map((title, i) => {
```

Comment:
нарушен критерий наименования
```suggestion
const allMovies = movieTitles.map((title, i) => {
```